portfolio/src/main/webapp/index.html

```diff
@@ -23,6 +23,9 @@
       <h1>Hey, hello!</h1><p>
       <img src="images/eyesGIF.png" width="700" height="525"/><p>
       <h2>Welcome to Micayla's Portfolio</h2>
+
+      <button onclick="getMessage()">Fetch a message!</button>
+      <div id="message-container"></div>
```

Comment:
Notice how this PR also has 2B content? That's because the approved PR hasn't been merged and then pulled back into your repo. Please merge the outstanding approved PRs and pull, this will allow me to review just the changes for this step.